test/Tests/src/System/Environment_Spec.enso

```diff
@@ -0,0 +1,34 @@
+from Standard.Base import all
+
+import Standard.Base.System.Environment
+import Standard.Test
+
+spec = Test.group "Environment" <|
```

Comment:
Think we need an `Environment.remove_overrides` to make sure Test starting in a clean state.

Reply:
Not really, the whole point of how the `Environment_Utils.with_environment_variable_override` (the only place to manipulate the overrides) works is that it sets the override and then unsets after the inner action finishes execution.

So, outside of a closure executed from within `unsafe_with_environment_override`, the overrides will never be set. So I don't think a reset method is necessary - its necessity would indicate a bug in the current override implementation.